src/astro/files/locations/base.py

```diff
@@ -0,0 +1,78 @@
+import glob
+import os
+from abc import ABC, abstractmethod
+from typing import List, Optional
+from urllib.parse import urlparse
+
+from astro.constants import FileLocation
+
+
+class Location(ABC):
+    """Base Location abstract class"""
+
+    def __init__(self, path: str, conn_id: Optional[str] = None):
+        """
+        Location class constructor
```

Comment:
This needs to be changed to something that explains what this is class is about  - "Location class constructor" doesn't tell much

Reply:
Done.